### PR TITLE
Merge Develop

### PR DESCRIPTION
**Pull request by vineet-amuratech**

### CODE REVIEW

File: sample_desktop.json [hunk 1]
```diff
@@ -0,0 +1,1030 @@
+{
```
Comment (chinmay-amuratech) [inline sample_desktop.json]: We don't need this file on master branch.

File: sample_mobile.json [hunk 2]
```diff
@@ -0,0 +1,1178 @@
+{
```
Comment (chinmay-amuratech) [inline sample_mobile.json]: We can remove this file.

File: sample_desktop.json (hunk 1 above)
Comment (vineet-amuratech) [inline sample_desktop.json]: We will have to remove these files from all branches then.

Comment (chinmay-amuratech) [inline sample_desktop.json]: Okay. I will update this in a while.

**Pull request merged**